packages/block-editor/src/components/dark-editor-style/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import tinycolor from 'tinycolor2';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+export default function DarkEditorStyle( ref ) {
+	useEffect( () => {
+		const { ownerDocument } = ref.current;
+		const editorStylesWrapper = ownerDocument.getElementsByClassName(
+			'editor-styles-wrapper'
+		);
+		const backgroundColor = window
+			.getComputedStyle( editorStylesWrapper[ 0 ], null )
+			.getPropertyValue( 'background-color' );
+
+		// Calculate the luminosity (0 - 255)
+		const brightness = tinycolor( backgroundColor ).getBrightness();
+
+		if ( brightness > 100 ) {
```

Comment:
It would be better if we used luminance here... Brightness of a color is absolute, while luminance depends on the hue and is closer to the perceived brightness.
Since the `brightness` const was only used in this one place, I think we can also remove it and just use the expression directly in the condition

```suggestion
		if ( tinycolor( backgroundColor ).getLuminance() > 0.5 ) {
```